packages/library/src/commonMain/kotlin/io/realm/RealmConfiguration.kt

```diff
@@ -17,58 +17,154 @@
 package io.realm
 
 import io.realm.internal.Mediator
+import io.realm.internal.PlatformHelper
 import io.realm.internal.RealmModelInternal
 import io.realm.internal.RealmObjectCompanion
 import io.realm.interop.NativePointer
 import io.realm.interop.RealmInterop
 import io.realm.interop.SchemaMode
+import io.realm.log.LogLevel
+import io.realm.log.RealmLogger
 import kotlin.reflect.KClass
 
-class RealmConfiguration(
-    val path: String? = null, // Full path if we don't want to use the default location
-    val name: String = "default", // Optional Realm name (default is 'default')
-    schema: Set<KClass<out RealmObject>> // classes literal (T::class)
+/**
+ * Configuration for log events created by a Realm instance.
+ */
+public data class LogConfiguration(
```

Comment:
When we enable Explicit API mode, it is going to be required, so I thought I just wanted to add it sooner rather than later: https://github.com/realm/realm-kotlin/issues/22

But if it confusing I can remove them again?

Reply:
Ah forgot about this one, let's keep it